_clients/data-prepper/data-prepper-reference.md

```diff
@@ -106,9 +106,10 @@ buffer_size | No | Integer | The maximum number of records the buffer accepts. D
 batch_size | No | Integer | The maximum number of records the buffer drains after each read. Default is 8.
 
 
-## Preppers
+## Processors
 
-Preppers perform some action on your data: filter, transform, enrich, etc.
+Processors perform some action on your data: filter, transform, enrich, etc.
+> Note: Prior to Data Prepper 1.3, Processors were named Preppers. Starting in Data Prepper 1.3, the term Prepper is deprecated in favor or Processor. Data Prepper will continue to support the term "Prepper" until 2.0 where it will be removed.
```

Comment:
We actually have some markdown classes you can use to call out notes here. You can use the following text instead:

Data Prepper versions prior to 1.3 used the term Prepper. Starting in Data Prepper 1.3, the term Prepper is deprecated in favor or Processor. Data Prepper will continue to support the term "Prepper" until 2.0, where it will be removed.
{: .note}